### DISCUSSION TITLE
Plotting Two Minerals on One Pole Figure

### DISCUSSION THREAD

**Discussion by beroshcr** [mtex-toolbox/mtex#1475] [Ask Anything] [answered]
Hello,

I am trying to plot the grain orientation for for two plagioclase feldspar minerals (albite, oligoclase) on to a singular pole figure.

Is there any way to store the grain data for both sets of minerals into one variable? 

Both Albite and Oligoclase are triclinic with a -1 symmetry operator. 

Thank you.

**Comment by kilir** [comment, marked as answer]
Hi,
you can compute the specimen directions for each pole figure (`r_ab_001 = o_ab * h_ab_001` and `r_olg_001 = o_olg * h_olg_001`) which are then of vector3d and you can plot those together in a pole polefigure.
Cheers,
Rüdiger

**Reply by beroshcr** [reply]
Hi Rüdiger,

Thank you very much. I have been able to compute vectors for both mineral orientations. 

However, what is the nomenclature/operation for combining two 3d vectors. My albite vector is 513 x 3 (100, 010, 001) and my oligoclase vector is 96 x 3. 

I want to combine the vectors to receive a 609 x 3 vector for plotting on the pole figure. 

I've tried:

r_Pl = [r_ab,r_ol];

and

r_Pl = r_ab + r_ol

But I am receiving errors. Where am I going wrong?

**Reply by kilir** [reply]
Hi,
`[ ]` should actually work, just make sure that the dimensions concatenated are fine. Otherwise just use something like 
```
v_001_ab = orientation_albite * Miller(1,0,0,cs_albite)
v_001_olig = orientation_olig * Miller(1,0,0,cs_olig)

plot([v_001_ab v_001_olig])
``` 

Cheers,
Rüdiger
